packages/notifier-seeder/.env

```
@@ -0,0 +1,8 @@
+LOG_LEVEL=info
+
+KAFKA_CLIENT_ID="notifier-seeder-client"
+KAFKA_GROUP_ID="notifier-seeder--group-local"
+KAFKA_BROKERS="localhost:9092"
+KAFKA_DISABLE_AWS_IAM_AUTH=true
+KAFKA_TOPICS="event-store.catalog.events"
```

based on [this PR](https://github.com/pagopa/interop-be-monorepo/pull/276)
```suggestion
CATALOG_TOPICS="event-store.catalog.events"
```

---

Sure 👍🏻, I'm still waiting to use `CATALOG_TOPICS` until [PR](https://github.com/pagopa/interop-be-monorepo/pull/276) was merged.

---

That PR has been merged ✅ 
The env name is `CATALOG_TOPIC` (no plural)